Scripts/script-SearchIncidents.yml

```diff
@@ -2,16 +2,44 @@ commonfields:
   id: SearchIncidents
   version: -1
 name: SearchIncidents
+deprecated: true
 script: |-
+  function invalideEscape(str) {
+      var i;
+      var special = ["n", "t"];
+      for (i = 0; i < str.length - 1; i++) {
+          if (str[i] === '\\') {
+              if (special.indexOf(str[i + 1] == -1)) {
+                  return str.substr(0, i);
+              }
+          }
+      }
+      return str;
+  }
+
+
+  // Clean all the empty ones
+  Object.keys(args).forEach(function(a) {if (!args[a]) delete args[a];});
+  var keys = Object.keys(args);
+  var i;
+  for (i = 0; i < keys.length; i++) {
+      temp = invalideEscape(args[keys[i]]);
+      if (temp.length != args[keys[i]].length) {
+          var md = "##### Error while parsing the argument: '" + keys[i] +
+          "'\n##### Succeeded parsing untill:\n- " + temp;
+          return {Type: entryTypes.error, Contents: md, ContentsFormat: formats.markdown, HumanReadable: md};
```

Comment:
If you have more than 1 invalid argument, you will onlny message the user about the first.
can you accumulate the *all* the invalid args, and return the error message for all of them in the end(only if there is an error)